testsuite/tests/libratest/smoke_test.rs

```diff
@@ -930,6 +930,72 @@ fn test_e2e_reconfiguration() {
     );
 }
 
+#[test]
+fn test_e2e_modify_publishing_option() {
+    let (mut env, mut client_proxy) = setup_swarm_and_client_proxy(1, 0);
+    client_proxy.create_next_account(false).unwrap();
+
+    client_proxy
+        .mint_coins(&["mintb", "0", "10"], true)
+        .unwrap();
+    assert_eq!(
```

Comment:
Do we need to test this?

Reply:
Minting is to prove regular transaction can be processed successfully. Do you think we need it?

Reply:
I don't think so--this should be covered by a simpler test that ensures that you can mint from the default configuration.